### ISSUE TITLE
LOG issue 

### ISSUE THREAD

**Issue by jzhanghzau** [Lightning-AI/pytorch-lightning#19794]
Bug description

As shown in the below's screenshot, 710144 is my total number of samples, but 100 is the batch amount. Since my batch size is 64, so I expect the total number is 710144/64=11096, I think 11096 should be at the position of 710144. Can someone explain me this? Which makes me a little bit confused.

This is the way I log during training step:

`self.log('train_loss', loss, on_step=True, rank_zero_only=True)'`

Thanks in advance!

JJ

**Comment by jzhanghzau** [comment]
`
self.log('train_loss', loss, on_step=True, rank_zero_only=True)

  self.trainer.strategy.barrier()`

**Comment by awaelchli** [comment]
The progress bar doesn't show the number of samples, but just the total number of batches (the length of the dataloader). You can confirm this by printing `len(train_dataloader)` in your training step.

In previous versions of Lightning, it also included the validation dataloader size. I'm not sure which version you are using here.

I'm not aware of any bugs here, so if you see something incorrect, could you please provide a code example?
